user/src/application.cpp

```diff
@@ -1,30 +1,26 @@
-/*
- * Copyright (c) 2019 Particle Industries, Inc.  All rights reserved.
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation, either
- * version 3 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHAN'TABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, see <http://www.gnu.org/licenses/>.
- */
-
-/* Includes ------------------------------------------------------------------*/
-#include "application.h"
-
-/* This function is called once at start up ----------------------------------*/
-void setup()
-{
-}
+#include "Particle.h"
+
+SYSTEM_MODE(MANUAL);
+
+SYSTEM_THREAD(ENABLED);
+
+Serial1LogHandler log(LOG_LEVEL_ALL);
 
-/* This function loops forever -----------------------------------------------*/
-void loop()
-{
-    // This will run in a loop
+void setup() {
```

Comment:
Let's also please not modify the default blank app.

Reply:
This was going to be an intermediate commit for testing purpose. I'll create a separated application then.